t/40-openqa-clone-job.t

```diff
@@ -21,14 +21,16 @@ use Test::Exception;
 use Test::More;
 use FindBin;
 use lib "$FindBin::Bin/lib";
-use OpenQA::Test::Utils qw(run_cmd test_cmd);
-
+use Date::Format 'time2str';
+use Mojo::File qw(path curfile tempfile);
+use OpenQA::Test::Utils qw(run_cmd test_cmd stop_service);
+use OpenQA::Test::Database;
```

Comment:
I recently saw this inconsistently handled in many files but I would prefer if we can group imports a little better, at least group by "external" and "internal" dependencies. Move every *external* dependency *above* the `use FindBin`, e.g.:

```perl
use Date::Format 'time2str';
use Mojo::File qw(path curfile tempfile);
use FindBin;
use lib "$FindBin::Bin/lib";
use OpenQA::Test::Utils qw(run_cmd test_cmd stop_service);
use OpenQA::Test::Database;
```

as we only need the `use lib…` to resolve all internal modules.